.github/workflows/publish-benchmark.yaml

```diff
@@ -0,0 +1,27 @@
+name: Publish Benchmark
```

Comment:
what is the difference between this and the `benchmark` in ci.yml?

Reply:
The same benchmark, but the result is published to github pages https://caraml-dev.github.io/universal-prediction-interface/dev/bench/